pages/events/index.vue

```diff
@@ -161,6 +187,26 @@ export default class extends QiskitPage {
     return (this as any).filteredEvents.length === 0
   }
 
+  getOptions (type) {
+    return this[type].map((item) => {
+      return {
+        label: item,
+        value: item,
+        name: item
+      }
+    })
+  }
+
+  updateFilters (filter: string, filterValues: object) {
+    const { commit } = this.$store
+    const payload = {
+      filter,
+      filterValues
+    }
+
+    commit('updateFilterSet', payload)
+  }
```

Comment:
I usually avoid naming two functions so similar (only adding an "s"). In this case `updateFilter` and `updateFilters`. I like to complete with a word like `updateFilterValues` or `updateWholeFilter`.

I would like to know @delapuente and @y4izus opinion here.

Reply:
I like the `updateWholeFilter` suggestion because the name expresses perfectly what it's doing